### PR TITLE
Depreciation of system_profile plugin

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

### Description

Created a deprecations page for the removal of the system_profile plugin per: https://github.com/chef/chef-web-docs/pull/1704. This PR needs to be amended to add an Ohai deprecations number in both files in this PR.

### Definition of Done

When @tas50 says it is okay.

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: chef_master/source/chef_deprecations_client.rst [hunk 1]
```diff
@@ -215,3 +215,7 @@ All Deprecations
     - Removal of IpScopes plugin
     - 13.2
     - 14.0
+  * - `OHAI-XX </depreciations_ohai_system_profile.html>`__
```
Comment (tas50) [inline chef_master/source/chef_deprecations_client.rst]: 14 probably

File: chef_master/source/chef_deprecations_client.rst [hunk 2]
```diff
@@ -215,3 +215,7 @@ All Deprecations
...
+    - Removal of system_profile plugin
+    - XX.X
```
Comment (tas50) [inline chef_master/source/chef_deprecations_client.rst]: 14.6

File: chef_master/source/depreciations_ohai_system_profile.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_ohai_system_profile.rst>`__
+
+.. tag depreciations_ohai_system_profile
+
+The system_profile plugin will be removed from Chef/Ohai 15 in April 2019. This plugin does not correctly return data on modern Mac systems. Additionally the same data is provided by the hardware plugin, which has a format that is simpler to consume. Removing this plugin will reduce Ohai return by ~3 seconds and greatly reduce the size of the node object on the Chef server.
+
+.. end_tag
+
+
+Example
```
Comment (tas50) [inline chef_master/source/depreciations_ohai_system_profile.rst]: The example bit can go away

File: chef_master/source/depreciations_ohai_system_profile.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+==========
+
+
+
+Remediation
```
Comment (tas50) [inline chef_master/source/depreciations_ohai_system_profile.rst:12]: Something something...

If you relied on data in node['system_profile'] you should look at the format in node['hardware'] and migrate to that new data format.

**Review by kagarmoe — APPROVED** [review]

**Comment by mjingle** [comment]
@tas50 Should we ship it?

**Review by tas50 — APPROVED** [review]

**Pull request merged**